source/versioned-api/tests/crud-api-version-1-strict.json

```diff
@@ -141,6 +141,11 @@
     },
     {
       "description": "aggregate on database appends declared API version",
+      "runOnRequirements": [
+        {
+          "minServerVersion": "4.9.0"
```

Comment:
I would just change minServerVersion to 4.9.0 on line 6 and remove this.  There is no reason any of these tests need to run on 4.7.0 anymore.

Reply:
If you don't mind I'd merge this as-is until we've sorted that topic.

Reply:
It's ok, but is there a rush to get this done?  I'd rather avoid churn.